### PR TITLE
ccw3398

### PR DESCRIPTION
**Pull request by nke001**
fix gh-3398

### CODE REVIEW

File: theano/compile/function_module.py [hunk 1]
```diff
@@ -1858,7 +1857,7 @@ def convert_function_input(input):
       `In`(r, name=name, value=val, update=up, autoname=True)
 
     """
-    if isinstance(input, (SymbolicInput, SymbolicInputKit)):
+    if isinstance(input, (SymbolicInput)):
```
Comment (nouiz) [inline theano/compile/function_module.py]: Remove the extra parenthesis.

File: theano/compile/function_module.py [hunk 2]
```diff
@@ -1887,7 +1886,7 @@ def convert_function_input(input):
             else:
                 raise TypeError("Invalid input syntax: %s (check "
                                 "documentation or use an In instance)" % orig)
-        elif isinstance(input[0], (SymbolicInput, SymbolicInputKit)):
+        elif isinstance(input[0], (SymbolicInput)):
```
Comment (nouiz) [inline theano/compile/function_module.py]: idem

File: theano/compile/function_module.py [hunk 3]
```diff
@@ -432,41 +430,41 @@ def distribute(indices, cs, value):
...
-                    finder[sin.name] = c
-                    if sin.name not in finder:
-                        finder[sin.name] = c
-                    else:
-                        finder[sin.name] = DUPLICATE
-                    inv_finder[c] = input
-                    c.required = required
-                    c.provided = 0
-                containers[:len(indices)] = []
+            # else:
```
Comment (nouiz) [inline theano/compile/function_module.py]: remove the code that you commented

File: theano/compile/function_module.py [hunk 4]
```diff
@@ -1033,16 +1031,16 @@ def _pickle_Function(f):
...
-            li = len(indices)
-            if not default:
-                input_storage.append(ins[:li])
-            else:
-                input_storage.append(default)
-            ins[:li] = []
-        else:
-            input_storage.append(ins[0])
-            del ins[0]
+        # if isinstance(input, SymbolicInputKit):
```
Comment (nouiz) [inline theano/compile/function_module.py]: idem

**Comment by nouiz** [comment]
ok to merge when travis pass

**Comment by nke001** [comment]
Ive pushed my changes, it is waiting in the queue for build.

https://travis-ci.org/Theano/Theano/builds/137321006

**Pull request merged**